circleci/fabfile.py

```diff
@@ -218,6 +218,12 @@ def _run_remote_lte_integ_test(repo: str, magma_root: str):
         # Copy to the directory CircleCI expects
         local('sudo mkdir -p /tmp/test-results/')
         local('sudo mv test-results/* /tmp/test-results/')
```

Comment:
Do you mean to replace this line with the new one?

Reply:
Yes @themarwhal. I thought @tmdzk was reviewing it but if you have questions, I am happy to answer it for you. Thank you for taking a look.